README.md

```diff
@@ -190,7 +190,7 @@ Feel free to check the [issues page][issues-url].
 
 Icon from:
 
-<a target="_blank" href="https://icons8.com/icons/set/credit-card">icon</a> by <a target="_blank" href="https://icons8.com">Icons8</a>
+<a target="_blank" href="https://icons8.com">Icons8</a>
```

Comment:
maybe move icons8 text to front of "Icon from:", i.e: 

Icon From: <a target="_blank" href="https://icons8.com">Icons8</a>

@Israel-Laguan @angelik what do you think?

Reply:
I think it should keep the original structure because other repositories also have it

Reply:
I am OK with either, I think Angelica should decide